### ISSUE TITLE
Signature spoofing does not work

### ISSUE THREAD

**Issue by GuzTech** [microg/android_packages_apps_GmsCore#210]
Dear developers,

I've got a BV6000 phone running their latest stock Android 6.0 ROM (22/08/16) that I've rooted. All the Google apps are removed, Xposed with framework SDK23 is installed, and FakeGApps 1.1 is also installed.

After installing MicroG Services Core (0.2.4-22-gcb356d2), Services Framework (0.1.0), and FakeStore (0.0.1), I've rebooted and when I do a self-check I see that everything except signature related entries are unchecked.

I did the same thing with my Moto G running CyanogenMod (Android 4.4) and that one works just fine. Is this because of Android 6 vs 4.4, or is it the stock vs CyanogenMod ROMs? I've done this before, but are the steps different for Android 6?

**Comment by ale5000-git** [comment]
I think your problem must be directed to the creator of FakeGApps module for Xposed.

If you want a permant support for signature spoofing you could also use this: https://github.com/ale5000-git/tingle

**Comment by GuzTech** [comment]
I'll ask the creator of FakeGApps too, but since I don't know where it goes wrong, I thought I'd mention it here.

/offtopic
Btw, I've tried your fork, but I get an error that no dex files were found, probably because the ROM is odexed (error code 86).

**Comment by ale5000-git** [comment]
@GuzTech: Yes, for it to working you need to deodex the ROM, I haven't yet found a different way.

**Comment by sorcer1122** [comment]
Hi guys,

I am also having this problem on EMUI (Huawei's version of Android 6.0.1). FakeGapps is installed but signature spoofing check box is empty. I have patched framework.jar using @ale5000-git method, put it back onto my phone with necessary permissions, reboot the phone....and nothing happened, there is still no sig spoofing check box.

Is it normal?

**Comment by Faerbit** [comment]
@bonanza123 said to report [this](https://github.com/thermatk/FakeGApps/issues/15] here. Is this related, or should I open a seperate issue?

**Comment by ale5000-git** [comment]
@Faerbit: Your problem is related to microG but not related to this issue, for your problem you should update [microG Services Core](https://microg.org/download.html)to the latest preview (currently 0.2.4-22, note the 22).

**Comment by ale5000-git** [comment]
@GuzTech: Any update on this issue?

**Comment by GuzTech** [comment]
@ale5000-git Unfortunately no. I've given it up. I wanted to use it for LibreSignal, but since it's not going to be supported by Signal, I dropped it and therefore have no use for it anymore :(

**Comment by ale5000-git** [comment]
Signal also works with microG.
